lib/services/product_brutto_price.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class ProductBruttoPrice
+    def call(basket_id, product_id)
+      product = FetchProductFromBasket.new.call(basket_id, product_id)
+      product.quantity * product.price * (1 + product.vat)
```

Comment:
@p55 could you specify, please? I've tried many combinations with spaces and it haven't worked...